src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -613,7 +615,7 @@ def __call__(
                 if not is_torch_available():
                     raise ImportError("Unable to convert output to PyTorch tensors format, PyTorch is not installed.")
 
-                encoded_inputs["target"] = [
+                encoded_inputs["labels"] = [
```

Comment:
Hmmm we'll probably need a patch release for this.
WDYT @sgugger?

Reply:
Yes, we should include it in the next patch release indeed.